### DISCUSSION TITLE
Incorrect Frame Order in VisIt Animation using Command from User's Guide

### DISCUSSION THREAD

**Discussion by JerryNiOnly** [AMReX-Codes/amrex#3363] [Q&A] [answered]
Hi, I would like to share my experience using VisIt to create movies based on plotfiles produced by AMReX

**Description**
I ran a simulation with more than 2 million steps (from t = 0 to t = 2.0) which generated piles of plotfiles. In order to create a movie to explicitly show the evolution process of the simulation, I refered to `User's Guide/Visualization/VisIt` and found that I can use the command introduced in the tutorial to create a file list which can be used by VisIt to create an animation. 
```console
ls -1 plt*/Header | tee movie.visit
```

**Behavior**
I expected that the animation could exhibit each frames ranging from t = 0 to t = 2.0 in order but things got messed up. The terminal frame exhibited t = 0.71 and the frame exhibiting t = 2.0 was near the middle of the animation. I didn't realise this problem until I was writing report 5 days after I created the movie. I tried to get the screenshot of the plot at t = 2.0 but I couldn't find it at the end of the animation.

**Cause and Solution**
It turned out that the command introduced in User's Guide can handle a small number of plotfiles but the `ls` command could mess up the order of plotfile when the number gets larger since it sorts files using an ACSII order. For example, 'plt269000' will be written ahead of 'plt27000'. So I managed to solve this problem by using the `sort` method to split the file name  to create a good `movie.visit` where plotfiles are in perfect order:
```console
ls -1 plt*/Header | sort -n -t "t" -k 2 | tee movie.visit
```

I hope this post could help VisIt users who may encounter similar problems which could possibly be ignored when creating movies. And I can't be more appreciated that if this could be included in the User's Guide to avoid misleading. 

Thank you and I hope you have a nice day!

**Comment by WeiqunZhang** [comment, marked as answer]
Thanks for reporting the issue! We will update the User's Guide.

I think `ls -1v` should also work.
```
-v     natural sort of (version) numbers within text
```
